src/js/api.ts

```diff
@@ -683,6 +683,24 @@ export class PyodideAPI {
     }
     return API.makeSnapshot(serializer);
   }
+
+  /**
+   * Returns the `info` section of the pyodide lockfile used to load the current Pyodide instance.
+   * The format of the lockfile is defined in the `pyodide/pyodide-lock
+   * <https://github.com/pyodide/pyodide-lock`_ repository.
+   */
+  static getLockfileInfo() {
```

Comment:
How about making these getters? 
```
   static get lockfileInfo() {
      return API.lockfile_info;
   }
```

Reply:
And maybe this could be `lockfilePlatform`? That seems a bit clearer to me.

Reply:
Sounds good to me. A minor downside is that the info section contains `platform` attribute. So it would be  `lockfilePlatform().platform` to access that attribute.